examples/armada-manifest-v1.yaml

```diff
@@ -125,41 +147,227 @@ data:
+              ovs:
+                agent:
+                  tunnel_types: vxlan
+        ovs:
+          neutron:
+            ml2:
+              ovs:
+                agent:
+                  of_interface: ovs-ofctl
+                  ovsdb_interface: vsctl
+                  bridge_mappings: public:br-ex
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm
+    subpath: neutron
+    reference: master
+  dependencies:
+    - helm-toolkit
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: nova
+data:
+  chart_name: nova 
+  release: nova 
+  namespace: openstack
+  timeout: 100
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  values:
+    ceph:
+      enabled: false
+    conf:
+      nova:
+        default:
+          oslo:
+            log:
+              debug: false
+        libvirt:
+          nova:
+            conf:
+              virt_type: qemu
+              images_type: null
+              images_rbd_pool: null
+              images_rbd_ceph_conf: null
+              rbd_user: null
+              rbd_secret_uuid: null
+              disk_cachemodes: null
+              hw_disk_discard: null
+        upgrade_levels:
+          nova:
+            conf:
+              compute: null
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm
+    subpath: nova
+    reference: master
+  dependencies:
+    - helm-toolkit
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: cinder
+data:
+  chart_name: cinder 
+  release: cinder 
+  namespace: openstack
+  timeout: 100
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  values: {}
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm
+    subpath: cinder
+    reference: master
+  dependencies:
+    - helm-toolkit
+---
 schema: armada/ChartGroup/v1
 metadata:
   schema: metadata/Document/v1
-  name: openstack-infra-services
+  name: deploy-mariadb
 data:
-  description: "OpenStack Infra Services"
+  description: "Mariadb"
   sequenced: True
   chart_group:
     - mariadb
+---
+schema: armada/ChartGroup/v1
+metadata:
+  schema: metadata/Document/v1
+  name: openstack-infra-services
+data:
+  description: "OpenStack Infra Services"
+  sequenced: False
+  chart_group:
     - etcd
     - rabbitmq
     - memcached
+    - ingress
 ---
 schema: armada/ChartGroup/v1
 metadata:
   schema: metadata/Document/v1
-  name: openstack-keystone
+  name: openstack-components
 data:
-  description: "Deploying OpenStack Keystone"
+  description: "OpenStack Components"
   sequenced: False
   chart_group:
     - keystone
+    - horizon
+    - glance
+    - neutron
+    - nova
+    - cinder
 ---
 schema: armada/Manifest/v1
 metadata:
   schema: metadata/Document/v1
   name: armada-manifest
 data:
-  release_prefix: armada
-  chart_groups:
-    - openstack-infra-services
-    - openstack-keystone
+  armada:
```

Comment:
@gardlt L368 adds a new level to the Manifest section which will probably break code somewhere. do we want to keep this and adjust code or remove it?